spec/requests/api/v1/articles/index_spec.rb

```diff
@@ -65,4 +75,28 @@
     end
   end
 
+  describe 'Gets index page by location' do
```

Comment:
what happens if there are no articles from that location or articles within in a specific category from that location?

Reply:
accounting for these scenarios now

Reply:
Saw something about a message in the client for that scenario but serving non-local articles together with a message that no local articles could be found is probably nicer